src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -8,12 +8,12 @@
 class Pokemon {
 
     private int pokemonId;
-    private String pokemonNumber;
+    private int pokemonNumber;
     private String pokemonName;
     private String pokemonImg;
     private List<String> pokemonType;
-    private String pokemonHeight;
-    private String pokemonWeight;
+    private double pokemonHeight;
+    private double pokemonWeight;
```

Comment:
Second Technical Suggestion: Changed some of the types for certain fields relating to numbers. int was used for those with solid numbers and no decimals while double was used for things that tend to have decimals.